core/data/tests/excluded_by_target_os/input.rs

```diff
@@ -32,3 +44,35 @@ pub enum Test {}
 #[cfg(feature = "super")]
 #[cfg(target_os = "android")]
 pub enum SomeEnum {}
+
+#[typeshare]
+#[cfg(any(target_os = "ios", taget_os = "android"))]
+pub struct ManyStruct;
```

Comment:
It looks like this type is not in the output files due to the bug with the ordering of the `target_os`.

`MultipleTargets` type is included because "android" is the first listed target, but this type is currently being skipped because "android" is the second listed target.

Reply:
It actually works. The input in my test is wrong. It says `taget_os` instead of `target_os`. I'll update the test.